services/apps/test/apps/service/apps/permissions_test.clj

```diff
@@ -0,0 +1,79 @@
+(ns apps.service.apps.permissions-test
+  (:use [apps.user :only [user-from-attributes]]
+        [clojure.test])
+  (:require [apps.service.apps :as apps]
+            [apps.service.workspace :as workspace]
+            [apps.test-fixtures :as tf]))
+
+(defn- create-user [i]
+  (let [username (str "testde" i)]
+    {:user       username
+     :first-name username
+     :last-name  username
+     :email      (str username "@mail.org")}))
+
+(defn- create-user-map []
+  (->> (take 10 (iterate inc 1))
+       (mapv (comp (comp (juxt (comp keyword :user) identity) create-user)))
```

Comment:
This is minor and possibly more about me understanding it than anything else: is there a reason for the extra `comp` with only one argument? Expanded for posterity:

```clojure
(mapv
    (comp ; <- this one has only one argument, the second `comp`
        (comp 
            (juxt
                (comp keyword :user)
                identity)
            create-user)))
```